app/components/collections/works_component.html.erb

```diff
@@ -5,7 +5,7 @@
       <th><span class="visually-hidden"><%= I18n.t 'collection.actions' %></span></th>
       <th width="10%"><%= I18n.t 'collection.owner' %></th>
       <th><%= I18n.t 'collection.deposit_status' %></th>
-      <th data-type="date" data-format="MMM D, YYYY" aria-sort="descending"><%= I18n.t 'collection.last_modified' %></th>
+      <th data-type="date" aria-sort="descending"><%= I18n.t 'collection.last_modified' %></th>
```

Comment:
data-format not necessary here.

Reply:
@justinlittman Removing this seems to have broken sorting by dates (see: #3592). When I put it back, date sorting works again. Can you recall why you removed this attr?

Reply:
I believe that according to the documentation, the date format we were using would be recognized, so `data-format` wasn't necessary. I thought I had tested, but apparently not successfully.

Reply:
You mean we're not the only software team that is challenged by documentation updates? Huh!